tickets/fixtures/moderation_groups.json

```diff
@@ -0,0 +1,51 @@
+[
+   {
+      "model":"auth.group",
+      "pk":3,
```

Comment:
some of your fixtures have null pk, some are set. what's the difference?

Reply:
only the new fixture I've made has `"pk":null,`, the "old" ones all have specific ID. when pk is set to null it assigns an ID at the time of creation. I think this works well because it does not assume a specific ID.

Reply:
Yes, I think it's good that we can let django create the PK when needed. I'd prefer that we have all items in the fixture with null pk.

Reply:
I set a number of `pk` to `null` wherever possible (in an easy way) f5e643998d1350c2a37c8d9a7c4ab2fe91397051

I'd agree in further optimizing fixtures to completely remove all non-null `pk`s.